translations/es/LC_MESSAGES/messages.po

```diff
@@ -223,7 +228,7 @@ msgstr ""
 
 #: /content/includes/home.yaml
 msgid "Building the future web, together."
-msgstr "Armando el futuro de la web, juntos."
+msgstr "Juntos, Armando el futuro de la web."
```

Comment:
Any possibility to get the last item for the menu translated?  Specifically "Brand Guidelines" on line 207.